.drone.yml

```diff
@@ -6532,7 +6532,7 @@ steps:
       - name: tools
         path: /tools
     environment:
-      TROTTLING_QUOTA_DISABLE: false
+      TROTTLING_QUOTA_DISABLE: true
```

Comment:
Typo here :) Should be `TROTTLING_QUOTA_DISABLE`. And in old node run.

Reply:
@binier 
ou, thank you, now it is good everywhere?